app/services/support_interface/offer_conditions_export.rb

```diff
@@ -36,31 +36,31 @@ def offers
   private
 
     def qualification_type(form)
-      qualifications(form).map(&:level).join(',')
+      qualifications(form).map(&:level).join(', ')
```

Comment:
Should this be 

```
 qualifications(form).map(&:qualification_type).join(', ')
```

If we use level it will only ever return one of these

`  enum level: {
    degree: 'degree',
    gcse: 'gcse',
    other: 'other',
  }`